rust-toolchain

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.40.0"
```

Comment:
Hmmm... I've never used a `rust-toolchain` file. Did you mean [`rust-toolchain.toml`](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file)? In any case, I don't think that does what I want. I don't want to build and develop on this version, I just want to have it be supported. I think CI is good enough for that. Otherwise, I want to use whatever my default toolchain is. (Usually latest nightly.)